components/blitz/src/ome/services/blitz/impl/OmeroMetadata.java

```diff
@@ -197,6 +197,8 @@ public String handleLsid(IObject obj) {
             Class<? extends IObject> k = obj.getClass();
             if (Annotation.class.isAssignableFrom(k)) {
                 k = Annotation.class;
+            } else if (Shape.class.isAssignableFrom(k)) {
```

Comment:
So this type of change will be needed for each class hierarchy that we introduce?